content/en/blog/falcosidekick-2-29-0/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Falcosidekick 2.29.0
+linktitle: Falcosidekick 2.29.0
+author: Thomas Labarussias
+date: 2024-07-02
+slug: falcosidekick-2-29-0
+images:
+  - /blog/falcosidekick-2-29-0/images/falcosidekick-featured.png
+tags: ["Falcosidekick","Release"]
+---
+
+Almost 1 year without a release of Falcosidekick, but the 2.29.0 is finally here. Thanks to all contributors for their patience, you made amazing contributions and we're happy to finally have them available for all users.
+
+Like for every releases, a small recap of the adoption. Falcosidekick continues to be adopted, even if the rate is not as high as before, but we're sure it will explode once again with this new fresh verson.
```

Comment:
```suggestion
Like for every releases, a small recap about its adoption. Falcosidekick continues to be adopted, even if the rate is not as high as before, but we're sure it will explode once again with this new fresh version.
```